language/stdlib/modules/Libra.move

```diff
@@ -572,14 +572,18 @@ module Libra {
     /// `MintCapability<CoinType>` and `BurnCapability<CoinType>` resources.
     public fun register_currency<CoinType>(
         account: &signer,
-        _: &Capability<RegisterNewCurrency>,
+        // DD refactor
+        // _: &Capability<RegisterNewCurrency>,
+        tc_account: &signer,
         to_lbr_exchange_rate: FixedPoint32,
         is_synthetic: bool,
         scaling_factor: u64,
         fractional_part: u64,
         currency_code: vector<u8>,
     ): (MintCapability<CoinType>, BurnCapability<CoinType>)
     acquires CurrencyRegistrationCapability {
+        // Privilege check.  TODO (dd): Error code.
+        assert(has_register_new_currency_privilege(tc_account), 99);
```

Comment:
Unrelated to this PR, but we really need to give those errors like `99` names and then divide them into categories. It seems currently they are fully random. This also allows us to do a better way to deal with aborts_if specs, because errors of a certain category (internal errors) we can ignore.

@sblackshear @tzakian

Reply:
Tim's working on that, I believe.